runtime.go

```diff
@@ -7,7 +7,7 @@ import(
 func NewRunTime(an string) (*Runtime){
 	return &Runtime{
 		AppName:an,
-		svc: service.NewManager("mongo"),
+		svc: service.NewManager("inmemory"),
```

Comment:
rather than hard-coding here. 
read from runtime as we do for `port` and default will `mongo`
But, best practices are read from the environment variable, for that you can use viper.
https://github.com/spf13/viper
FYI it will not be part of this pull request.

Reply:
Okay Sir, will try this.